MapboxNavigation/NavigationViewController.swift

```diff
@@ -34,6 +34,9 @@ public protocol NavigationViewControllerDelegate {
     @objc(navigationViewController:shouldRerouteFromLocation:)
     optional func navigationViewController(_ navigationViewController: NavigationViewController, shouldRerouteFrom location: CLLocation) -> Bool
     
+    @objc(navigationViewController:shouldIncrementLegWhenArrivingAtWaypoints:)
+    optional func navigationViewController(_ navigationViewController: NavigationViewController, shouldIncrementLegWhenArrivingAtWaypoints waypoint: Waypoint) -> Bool
```

Comment:
I’m confused by the `shouldIncrementLegWhenArrivingAtWaypoints` argument: it makes the method sound like there’s only one setting for all waypoints in the route, yet this method is called once for each waypoint. Did you mean to make the argument name singular, i.e. `shouldIncrementLegWhenArrivingAtWaypoint`? If so, the Swift name should be `navigationViewController(_:shouldIncrementLegWhenArrivingAt:)` while the Objective-C name would be `-navigationViewController:shouldIncrementLegWhenArrivingAtWaypoint:`.